### ISSUE TITLE
Is support needed for non-linear digitalisation?

### ISSUE THREAD

**Issue by paulmillar** [nexusformat/definitions#1400]
Just as an observation, I believe NXdata describes a linear model for mapping between a continuous/observed values and the discrete/digitised signal.

A linear mapping is not the only approach.  Other approaches have been tried (in other fields) when it is desirable to focus more of the fidelity of digitised dynamic range in particular areas of interest; for real-world examples, see the [A-law](https://en.wikipedia.org/wiki/A-law_algorithm) and [μ-law](https://en.wikipedia.org/wiki/%CE%9C-law_algorithm) algorithms.

I am wondering if there is a need for NeXus to describe non-linear digitalisation?

_Originally posted by @paulmillar in https://github.com/nexusformat/definitions/pull/1343#discussion_r1701929911_

**Comment by phyy-nx** [comment]
I believe #711 is relevant here

**Comment by 7ScriptsGL** [comment]
Yes.  In the linear constructs the data being flattened tends to alert fails in one repo i been working on.  I been working it out.  But It is still a necessity in the approach because i havent achieve stable bells that self resolve as wll.
